includes/class-wc-data-structurer.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Structured data's handler and generator using JSON-LD format.
+ *
+ * @class 		WC_Data_Structurer
+ * @version		2.7.0
+ * @package		WooCommerce/Classes
+ * @category	Class
+ * @author 		Clement Cazaud
+ */
+class WC_Data_Structurer {
```

Comment:
Change the name of the method `WC_Structured_Data->get_data()`.
If the class control the "Structured Data", you should expect return "Structured Data" :P